content/redirects.txt

```diff
@@ -119,7 +119,6 @@
 
 # Extending Terraform section
 /guides/writing-custom-terraform-providers.html          /docs/extend/writing-custom-providers.html
-/community.html                                          /docs/extend/community/index.html
```

Comment:
As mentioned above in this document we'll need to coordinate the removal of this redirect with engserv team.